frontend/benefit/applicant/src/components/applications/forms/application/step5/employeeView/EmployeeView.tsx

```diff
@@ -128,13 +129,17 @@ const EmployeeView: React.FC<EmployeeViewProps> = ({
           <$ViewField>
             {t(`${translationsBase}.employee.fields.startDate.label`)}
           </$ViewField>
-          <$ViewField>{data.startDate ? data.startDate : '-'}</$ViewField>
+          <$ViewField>
+            {data.startDate ? formatDate(new Date(data.startDate)) : '-'}
```

Comment:
`{convertToUIDateFormat(data.startDate) || '-'}` should do the same?

Reply:
ok, didnt know that we have the helper, i ll fix